crawl-ref/source/dat/des/variable/mini_monsters.des

```diff
@@ -3060,6 +3082,33 @@ xGD..D..1..E..EGx
 xxxxxxxxxxxxxxxxx
 ENDMAP
 
+# Another NetHack-inspired vault, based on the top floor of Vlad's Tower.
+NAME:  nicolae_nh_vlads_tower
+TAGS:  transparent patrolling
+DEPTH: D:10-, Crypt
+KITEM: | = |
+KITEM: _ = *
+KITEM: 1 = %
+KMONS: _ = vampire mage / vampire knight
+KMONS: 1 = vampire / jiangshi
```

Comment:
this needs some depth conditioning for the proposed spawn range. Jiangshi and Vampire Knights are too much for D:11 en masse.

Reply:
> this needs some depth conditioning for the proposed spawn range. Jiangshi and Vampire Knights are too much for D:11 en masse.

pfffft, shouldna been standin there

(I'll make the changes.)